lib/graphql/analysis/ast/visitor.rb

```diff
@@ -222,6 +222,11 @@ def directive_definition
         def argument_definition
           # Don't get the _last_ one because that's the current one.
           # Get the second-to-last one, which is the parent of the current one.
```

Comment:
Does this comment need to change / be removed?

Reply:
Yep 👀 removed